app/models/bar.rb

```diff
@@ -1,4 +1,6 @@
 class Bar < ActiveRecord::Base
+  # has_many :reviews, dependent: :destroy
```

Comment:
Yes; we don't have a Review model in this branch, so that seemed out of scope, but we left it there for when we do? Should we have added the model anyway?